app/src/main/java/org/oppia/android/app/home/HomeActivity.kt

```diff
@@ -96,7 +97,7 @@ class HomeActivity :
     val recentlyPlayedActivityParams =
       RecentlyPlayedActivityParams
         .newBuilder()
-        .setProfileId(ProfileId.newBuilder().setInternalId(internalProfileId).build())
+        .setProfileId(ProfileId.newBuilder().setInternalId(profileId.internalId).build())
```

Comment:
Can just pass in profileId here.

Reply:
Thanks @BenHenning noted and updated, don't know how i missed this.

Reply:
Yeah updated this.